website/identifiers/metadata.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
```

Comment:
What's this file's story?

Reply:
This just wasn't removed with other EZid code.